android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageUiState.kt

```diff
@@ -0,0 +1,20 @@
+package org.uwaterloo.subletr.pages.createaccount
+
+sealed interface CreateAccountPageUiState {
+	object Loading : CreateAccountPageUiState
+
+	enum class Gender(val gender: String) {
+		MALE("Male"),
+		FEMALE("Female"),
+		OTHER("Other"),
+	}
+
+	data class NewAccountInfo(
```

Comment:
I mean, not the worst thing in the world, but I think `Loaded` is a better name.
Since the children of `CreateAccountPageUiState` are either `Loading` (meaning that we're waiting for something in the stream) or `Loaded` (which means we have received whatever we needed from the stream).

I feel like `NewAccountInfo` could be a good subclass name for a loaded data class but the loaded data class is essentially just state for `NewAccountInfo` stuff.